datasets/imagenet/README.md

```diff
@@ -0,0 +1,1221 @@
+
+### Source Data
+
+#### Initial Data Collection and Normalization
+
+Initial data for ImageNet image classification task consists of photographs collected from [Flickr](https://www.flickr.com) and other search engines, manually labeled with the presence of one of 1000 object categories. Constructing ImageNet was an effort to scale up an image classification dataset to cover most nouns in English using tens of millions of manually verified photographs [1](https://ieeexplore.ieee.org/abstract/document/5206848). The image classification task of ILSVRC came as a direct extension of this effort. A subset of categories and images was chosen and fixed to provide a standardized benchmark while the rest of ImageNet continued to grow.
+
+#### Who are the source language producers?
+
+WordNet synsets further quality controlled by human annotators. The images are from Flickr.
+
+### Annotations
+
+#### Annotation process
+
+The annotation process of collecting ImageNet for image classification task is a three step process.
+
+1. Defining the 1000 object categories for the image classification task. These categories have evolved over the years.
+1. Collecting the candidate image for these object categories using a search engine.
+1. Quality control on the candidate images by using human annotators on Amazon Mechanical Turk (AMT) to make sure the image has the synset it was collected for.
+
+See the section 3.1 in [1](https://arxiv.org/abs/1409.0575) for more details on data collection procedure and [2](https://ieeexplore.ieee.org/abstract/document/5206848) for general information on ImageNet.
+
+#### Who are the annotators?
+
+Images are automatically fetched from an image search engine based on the synsets and filtered using human annotators on Amazon Mechanical Turk. See [1](https://arxiv.org/abs/1409.0575) for more details.
+
+### Personal and Sensitive Information
+
+See the papers: [1](https://arxiv.org/abs/1409.0575) [2](https://ieeexplore.ieee.org/abstract/document/5206848).
```

Comment:
I think we should add a comment that the images contain faces of people who did not consent to this usage, which is why a recent version of ImageNet has face obfuscation applied, which does not impact results: https://arxiv.org/abs/2103.06191